### DISCUSSION TITLE
Does rslib need a configuration file in the root directory to be executed? Can it be run directly from a command?

### DISCUSSION THREAD

**Discussion by a145789** [web-infra-dev/rslib#512] [General]
use tsup, you can directly package 

```shell 
tsup src/index.ts --format esm --out-dir=lib --dts --clean 
``` 
Using rslib, it gives an error if there is no configuration file 

``` 
rslib.config not found in D:// 
```

Is it possible to use command line configuration for simple projects like `tsup`?

**Comment by Timeless0911** [comment]
Yeah, we will provide simple way to use Rslib without configuration file in the future after we implement all core features, tracking in https://github.com/web-infra-dev/rslib/issues/454.
